### ISSUE TITLE
Doesn't respect path prefix

### ISSUE THREAD

**Issue by SteveEdson** [thetre97/gridsome-plugin-flexsearch#17]
If the `gridsome.config.js` has a path prefix, such as

```javascript
module.exports = {
  siteName: 'Search Example',
  pathPrefix: '/gridsome',
};
```

The option is ignored, and the `flexsearch.json` file is still requested from the root of the site.

Also, from a quick look at the json file, it looks like the paths in the results don't have the prefix either.

**Comment by thetre97** [comment]
Thanks for raising this @SteveEdson - I hadn't thought about it before.
I have just published `0.1.11` which should add support for this. Would you be able to try this version, to check it works?

**Comment by SteveEdson** [comment]
Yep, perfect! 

Thanks for the quick response

**Comment by SteveEdson** [comment]
Sorry to re-open this, I've just noticed it still uses the pathPrefix when using `gridsome develop`, it only needs to use it in production build mode, and the develop mode ignores pathPrefix.

Thanks

**Comment by thetre97** [comment]
Could I ask if you are using the latest version (`0.1.12`)? I recently pushed an update to fix this (noted in issue #19 )

**Comment by SteveEdson** [comment]
Ah yeah, perfect. Thanks!
